pkg/auditd/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:edge AS mirror
```

Comment:
I think we don't get exact binary reproducibility with this method, unless the code base happens to support it, in general we only get the same functional output given the same inputs i.e. without variation in components due to network mirrors etc changing. (I suspect you are each talking about a different level of "reproducible" here).